test/ibmq/utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test for the qiskit-ibmq-provider package."""
```

Comment:
We don't usually create separate packages for tests, unless the functionality requires multiple test files (e.g. `runtime`). In addition, `test_serialization.py` already has some test cases for `IQXJsonEncoder`, so you can just reuse it.